src/App.tsx

```diff
@@ -15,48 +17,104 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum TypeOfSort {
+  name = 'name',
+  length = 'length',
```

Comment:
```suggestion
  NAME = 'name',
  LENGTH = 'length',
  NONE = 'none',
```
general convention make fields of enums in upper case
and better to set for none sort